tests/telemetry/test_telemetry.py

```diff
@@ -133,3 +135,54 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, localhost):
     result = str(show_gnmi_out)
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
+
+def test_sysuptime(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """
+    @summary: Run pyclient from ptfdocker and test the dataset 'system uptime' to check
+              whether the value of 'system uptime' was an integer and whether the value was
+              updated correctly.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    docker_present = verify_telemetry_dockerimage(duthost)
+    if not docker_present:
+        pytest.skip("docker-sonic-telemetry is not part of the image")
```

Comment:
Since every test case need to check the presence of telemetry docker image, can you make it a module level autouse fixture? Then the individual test cases do not need to care about this check.

Reply:
Great suggestion! Will update.